ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+        until $(curl -XGET https://localhost:9200/ -uadmin:admin -k --max-time 120 --silent --output /dev/null); do
+            echo -ne $char
+            sleep 10
+        done
+
+        cd /usr/share/elasticsearch/plugins/opendistro_security/tools/ $debug
+        ./securityadmin.sh -cd ../securityconfig/ -nhnv -cacert /etc/elasticsearch/certs/root-ca.pem -cert /etc/elasticsearch/certs/admin.pem -key /etc/elasticsearch/certs/admin.key $debug
+
+        echo "Done"
+    fi
+}
+
+## Filebeat
+installFilebeat() {
+    logger "Installing Filebeat..."
+
+    $sys_type install filebeat-"${ELK_VERSION}" -y -q  $debug
+    if [ "$?" != 0 ]; then
+        echo "Error: Filebeat installation failed"
+        exit 1;
+    else
+        WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+
+        curl -so /etc/filebeat/filebeat.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/filebeat/7.x/filebeat_all_in_one.yml --max-time 300  $debug
+        curl -so /etc/filebeat/wazuh-template.json https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json --max-time 300 $debug
+        chmod go+r /etc/filebeat/wazuh-template.json $debug
+        
+        curl -s https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz --max-time 300 | tar -xvz -C /usr/share/filebeat/module $debug
+        mkdir -p /etc/filebeat/certs $debug
+        cp /etc/elasticsearch/certs/root-ca.pem /etc/filebeat/certs/ $debug
+        mv /etc/elasticsearch/certs/filebeat.key /etc/filebeat/certs/ 
+        mv /etc/elasticsearch/certs/filebeat.pem /etc/filebeat/certs/ 
+        # Start Filebeat
+        startService "filebeat"
+        logger "Done"
+    fi
+}
+
+## Kibana
+installKibana() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Installing Open Distro for Kibana..."
+    $sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q $debug
+    if [ "$?" != 0 ]; then
+        echo "Error: Kibana installation failed"
+        exit 1;
+    else
+        curl -so /etc/kibana/kibana.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/kibana/7.x/kibana_all_in_one.yml --max-time 300 $debug
+        echo "telemetry.enabled: false" >> /etc/kibana/kibana.yml
+        cd /usr/share/kibana $debug
+
+        if [ "${STATUS_PACKAGES}" = "prod" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/${WAZUH_MAJOR}.x/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}.zip $debug
+        elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-1.zip $debug
+        fi
+                
+        if [ "$?" != 0 ]
+        then
+            echo "Error: Wazuh Kibana plugin could not be installed."
```

Comment:
Use `logger` function.

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2